capi/capi.cpp

```diff
@@ -46,14 +46,16 @@ class FileProcessor final
     open_options.ignore_sizem_for_pyramid_subblocks = true;
     reader->Open(stream, &open_options);
 
-    // create the "CZI-writer"-object
+    // create (and configure) the "CZI-writer"-object
     const std::string output_string(output_path);
     const auto output_stream = libCZI::CreateOutputStreamForFile(utils::utf8::WidenUtf8(output_string).c_str(), false);
 
+    libCZI::CZIWriterOptions czi_writer_options;
```

Comment:
Can we use designated initialization here?

`libCZI::CZIWriterOptions` czi_writer_options {
  .allow_duplicate_subblocks = true
};`

Reply:
I tried this, but this is a [C++20-feature](https://www.cppstories.com/2021/designated-init-cpp20/) (and so far we are using C++17).

Reply:
Ah, I got my wires crossed. I thought it was available earlier.